src/test/java/salesforce/assets/CreateAssetTest.java

```diff
@@ -11,52 +11,75 @@
+    public void testCreateAssetWithNecessaryAttributes() {
+        NewAssetPage createAssetPage = assetPage.clickCreateAssetBtn();
+        createAssetPage.setField("Asset Name", "Name Asset 1")
+                .clickField("Account")
+                .clickRoleFirstOption();
+        assetDetailPage = createAssetPage.clickSaveBtn();
+        softAssert.assertEquals(assetDetailPage.getField("Account"), "cuenta 13",
+                "Account name does not match");
+        Assert.assertEquals(assetDetailPage.getCreatedAssetTitleText(), "Name Asset 1",
                 "Asset was not created");
     }
 
     @Test
     public void testCreateAssetWithAllAttributes() {
-        CreateAssetPage createAssetPage = assetPage.clickCreateAssetBtn();
-        createAssetPage.setUserName("Name Asset 1");
-        createAssetPage.setSerialNumber("Serial Number");
-        createAssetPage.setQuantity("10");
-        createAssetPage.setPrice("100");
-        createAssetPage.setDescriptionTxtBox("Description");
-        createAssetPage.setInstallDateCalendar("8/7/2021");
-        createAssetPage.setPurchaseDateCalendar("15/7/2021");
-        createAssetPage.setUsageDateCalendar("25/7/2021");
-        createAssetPage.clickActive();
-        createAssetPage.clickStatus();
-        createAssetPage.clickStatusOptions();
-        createAssetPage.clickRoleOption();
-        createAssetPage.clickRoleFirstOption();
-        createAssetPage.clickContactOption();
-        createAssetPage.clickContactFirstOption();
-        createAssetPage.clickProductOption();
-        createAssetPage.clickProductFirstOption();
-        createdAssetPage = createAssetPage.clickSaveBtn();
-        Assert.assertEquals(createdAssetPage.getCreatedAssetTitleText(), "Name Asset 1",
+        NewAssetPage createAssetPage = assetPage.clickCreateAssetBtn();
+        createAssetPage.setField("Asset Name", "Name Asset 1")
+                .setField("Serial Number", "Serial Number")
+                .setField("Quantity", "10")
+                .setField("Price", "100")
+                .setDescription("Description")
+                .setField("Install Date", "8/7/2021")
+                .setField("Purchase Date", "15/7/2021")
+                .setField("Usage End Date", "25/7/2021")
+                .clickActive()
+                .clickStatus()
+                .clickStatusOptions()
+                .clickField("Account")
+                .clickRoleFirstOption()
+                .clickField("Contact")
+                .clickContactFirstOption()
+                .clickField("Product")
+                .clickProductFirstOption();
+        assetDetailPage = createAssetPage.clickSaveBtn();
+        softAssert.assertEquals(assetDetailPage.getField("Account"), "cuenta 13",
+                "Account name does not match");
+        softAssert.assertEquals(assetDetailPage.getField("Contact"), "contact 2",
+                "Account name does not match");
```

Comment:
each message should be different considering it is a different field.

Reply:
You were right. I already changed it.